src/pages/components/textinput.tsx

```diff
@@ -15,90 +15,105 @@ import { TextInput, suomifiDesignTokens } from 'suomifi-ui-components';
 
 TextInput.displayName = 'TextInput';
 
-const Page = (): JSX.Element => (
-  <Translation ns={['textinput']}>
-    {(t) => (
-      <Layout sideNavData={sideNavData(t)}>
-        <SEO title={t('title')} />
-        <Heading variant="h1">{t('title')}</Heading>
+const Page: React.FC = (): React.ReactElement => {
+  const [errorState, setErrorState] = React.useState(true);
+  const statusText = (t: Function): string => {
```

Comment:
This could be written with ternary operator and shown in the example. Could be more clear that way:
`statusText={errorState ? t('exampleError.statusText') : null}`
or even
`{...(errorState ? {statusText: t('exampleError.statusText')} : {})}`
Both should be ok after gatsby build.

Reply:
Good point. Used the latter option since it produced the cleanest output.